packages/react-scripts/scripts/init.js

```diff
@@ -43,12 +43,12 @@ module.exports = function(
     eject: 'react-scripts eject',
   };
 
-  appPackage.browserslist = [
-    '>1%',
-    'last 4 versions',
-    'Firefox ESR',
-    'not ie < 9',
-  ];
+  appPackage.browserslist = {
+    development: ['chrome', 'firefox', 'edge'].map(
+      browser => `last 2 ${browser} versions`
```

Comment:
Why this change? Does this correspond to any specific list we use elsewhere?

Reply:
Never mind, I got it.
Wish there was a shorter way to write this.
Maybe something like `>5%` is equivalent?

Reply:
I couldn't find a magic percent -- and I didn't want to discriminate browsers.